app/src/main/java/org/mozilla/fenix/compose/NoCollectionsMessage.kt

```diff
@@ -0,0 +1,157 @@
+                .padding(all = 16.dp)
+                .fillMaxWidth()
+        ) {
+            Row(
+                modifier = Modifier.fillMaxWidth(),
+                horizontalArrangement = Arrangement.SpaceBetween,
+                verticalAlignment = Alignment.CenterVertically,
+            ) {
+                SectionHeader(
+                    text = stringResource(R.string.collections_header),
+                    modifier = Modifier
+                        .weight(1f)
+                        .wrapContentHeight(align = Alignment.Top)
+                )
+
+                IconButton(
+                    onClick = { onRemoveNoCollectionsMessage() }
+                ) {
+                    Icon(
+                        painter = painterResource(R.drawable.ic_close),
+                        contentDescription = stringResource(
+                            R.string.remove_home_collection_placeholder_content_description
+                        ),
+                        modifier = Modifier.size(24.dp),
+                        tint = FirefoxTheme.colors.iconPrimary
+                    )
+                }
+            }
+
+            PrimaryText(
+                text = stringResource(R.string.no_collections_description2),
+                modifier = Modifier.fillMaxWidth(),
+                fontSize = 14.sp
+            )
+
+            if (showAddToCollectionButton) {
+                Spacer(modifier = Modifier.height(12.dp))
+
+                AddToCollectionButton(
+                    onClick = onAddTabsToCollectionClick
+                )
+            }
+        }
+    }
+}
+
+/**
+ * No Collections Tabs Button
+ *
+ * @param onClick The lambda called when the button is clicked.
+ */
+@Composable
+fun AddToCollectionButton(onClick: () -> Unit) {
+    Button(
+        onClick = onClick,
+        modifier = Modifier
+            .clip(RoundedCornerShape(size = 4.dp))
+            .fillMaxWidth(),
+        elevation = ButtonDefaults.elevation(defaultElevation = 0.dp, pressedElevation = 0.dp),
+        colors = ButtonDefaults.outlinedButtonColors(backgroundColor = FirefoxTheme.colors.actionPrimary),
+    ) {
+        Icon(
+            painter = painterResource(R.drawable.ic_tab_collection),
+            contentDescription = null,
+            tint = FirefoxTheme.colors.iconOnColor,
+        )
+
+        Spacer(modifier = Modifier.width(8.dp))
+
+        Text(
+            text = stringResource(R.string.tabs_menu_save_to_collection1),
+            modifier = Modifier.align(Alignment.CenterVertically),
+            color = FirefoxTheme.colors.textOnColorPrimary,
+            fontSize = 14.sp,
+            fontFamily = FontFamily(Font(R.font.metropolis_semibold)),
+            maxLines = 2
+        )
+    }
+}
+
+@Composable
+@Preview
+private fun NoCollectionsMessagePreview() {
+    NoCollectionsMessage(
+        true,
+        onAddTabsToCollectionClick = {},
+        onRemoveNoCollectionsMessage = {}
+    )
+}
```

Comment:
When testing this locally, the preview wasn't rendering without wrapping it with `FirefoxTheme`.

```suggestion
private fun NoCollectionsMessagePreview() {
    FirefoxTheme(theme = Theme.getTheme(isPrivate = false)) {
        Box(Modifier.background(FirefoxTheme.colors.layer1)) {
            NoCollectionsMessage(
                true,
                onAddTabsToCollectionClick = {},
                onRemoveNoCollectionsMessage = {}
            )
        }
    }
}
```